common/errors.js

```diff
@@ -6,12 +6,12 @@
     // Factory for each error type
     .factory('ErrorService', ['$log', '$window', '$uibModal', 'UriUtils', 'AlertsService', 'Session', function ErrorService($log, $window, $uibModal, UriUtils, AlertsService, Session) {
 
-        function errorPopup(exception) {
-            $log.info(exception);
+        function errorPopup(message, errorCode, pageName, redirectLink) {
 
```

Comment:
I made the changes to record-edit as well. I also factored it out in `recordedit.app.js`, but not here.